robot_specific_config/abb_irb1200_support/meshes/irb1200_7_70/visual/link_2.dae

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We don't support this robot, but it's theoretically not wrong. Should we remove this file + the other files in the irb1200_7_70 folder?